docs/tutorials/fundamentals/viewer.md

```diff
@@ -276,46 +251,30 @@ viewer.add_shapes()
+Layers can be deleted by selecting them and clicking on the trash icon, or by dragging the layers and dropping them into the trash.
 
 In the console a layer at index `i` can be removed by:
 
-```python
+```{code-cell} python
 viewer.layers.pop(i)
 ```
 
 +++
 
 ## Dimension sliders
 
-One of the main strengths of **napari** is that it has been designed from the
-beginning to handle n-dimensional data. While much consumer photography is 2D
-and `RGB`, scientific image data can often be volumetric (i.e. 3D), volumetric
-timeseries (i.e. 4D), or even higher dimensional. **napari** places no limits on
-the dimensionality of its input data for all its layer types.
+One of the main strengths of **napari** is that it has been designed from the beginning to handle n-dimensional data. While much consumer photography is 2D and `RGB`, scientific image data can often be volumetric (i.e. 3D), volumetric timeseries (i.e. 4D), or even higher dimensional. **napari** places no limits on the dimensionality of its input data for all its layer types.  
 
-Adding data with a dimensionality greater than 2D will cause dimension sliders
-to appear directly underneath the canvas and above the status bar. As many
-sliders as needed will appear to ensure the data can be fully browsed. For
-example, a 3D dataset needs one slider, a 4D dataset needs two sliders, and so
-on. The widths of the scroll bars of the dimension sliders are directly related
-to how many slices are in each dimension.
+Adding data with a dimensionality greater than 2D will cause dimension sliders to appear directly underneath the canvas and above the status bar. As many sliders as needed will appear to ensure the data can be fully browsed. For example, a 3D dataset needs one slider, a 4D dataset needs two sliders, and so on. The widths of the scroll bars of the dimension sliders are directly related to how many slices are in each dimension.
 
-It is also possible to mix data of different shapes and dimensionality in
-different layers. If a 2D and 4D dataset are both added to the viewer then the
-sliders will only affect the 4D dataset and the 2D dataset will be remain the
-same. Effectively, the two datasets are broadcast together using [NumPy
-broadcasting rules](https://numpy.org/doc/stable/user/basics.broadcasting.html).
+It is also possible to mix data of different shapes and dimensionality in different layers. If a 2D and 4D dataset are both added to the viewer then the sliders will affect only the 4D dataset, the 2D dataset will remain the 
+same. Effectively, the two datasets are broadcast together using [NumPy broadcasting rules](https://numpy.org/doc/stable/user/basics.broadcasting.html).  
 
-For example, the following commands from the console will add a both 2D and 3D
-datasets to the same viewer:
+For example, the following commands from the console will add both 2D and 3D datasets to the same viewer:
 
-```{code-cell} python
-:tags: [remove-output]
+```{code-cell} python  
```

Comment:
Should the tag be removed here? It's present in the other example above (astronaut & points)

Reply:
Removed 
:tags: [remove-output]
```{code-cell} python

Reply:
It doesn't seem to matter as far as the image is concerned.  I'm meeting with @melissawm tomorrow.  Maybe we can figure it out.